pkg/ccl/backupccl/backup_planning.go

```diff
@@ -886,39 +886,64 @@ func backupPlanHook(
 		}
 
 		collectTelemetry := func() {
-			telemetry.Count("backup.total.started")
+			// sourceSuffix specifies if this schedule was created by a schedule.
+			sourceSuffix := ""
+			if backupStmt.CreatedByInfo != nil &&
+				backupStmt.CreatedByInfo.Name == jobs.CreatedByScheduledJobs {
+				sourceSuffix = ".scheduled"
+			}
+
+			// countSource emits a telemetry counter and also adds a ".scheduled"
+			// suffix if the job was created by a schedule.
+			countSource := func(feature string) {
+				telemetry.Count(feature + sourceSuffix)
```

Comment:
hm. I wonder if we should keep the global counts for all backups -- scheduled or manual -- and have an additional counter for scheduled? otherwise I wonder if it is prone to forgetting to sum the two counters any time we look at this. I guess a question for @mwang1026

Reply:
Subtraction is hard in Looker but addition is easy. If we did go with a global counter, I'd want separate ones for manual and scheduled as well. Thinking through, I personally prefer the latter -- a global counter, and then a split one with manual and scheduled separate. That way dashboards won't break as we transition versions.

Reply:
Just chatted with @mwang1026, it sounds like we can search by a prefix. I think it would be worthwhile updating the default prefix to ".manual" so that there is a way to easily filter telemetry by scheduled and non-scheduled backups.

Put out https://github.com/cockroachdb/cockroach/pull/54952 for this change.